spec/services/reporting/account_reuse_report_spec.rb

```diff
@@ -19,10 +19,12 @@
     let(:sp_a) { 'a' }
     let(:sp_b) { 'b' }
     let(:sp_c) { 'c' }
+    let(:sp_d) { 'd' }
 
-    def create_identity(id, provider, verified_time)
+    def create_identity(id, created_time, provider, verified_time)
```

Comment:
1. I'd consider switching these to keyword arguments for clarity, then in the loop below we could just call `create_identity(**params)` instead of having to split out each part of the hash
2. I think we might want to rename `id` to `user_id` here, again to make sure it's clearer what's being passed